OptimizelySDK/Event/Builder/EventBuilder.cs

```diff
@@ -74,16 +78,24 @@ private static long SecondsSince1970
         /// <param name="config">ProjectConfig Configuration for the project</param>
         /// <param name="userId">string ID of user</param>
         /// <param name="userAttributes">associative array of Attributes for the user</param>
-        public void SetCommonParams(ProjectConfig config, string userId, UserAttributes userAttributes)
+        public Dictionary<string, object> SetCommonParams(ProjectConfig config, string userId, UserAttributes userAttributes)
```

Comment:
Rename this method, GetCommonParams. And plz. chk it should be private, i think.

Reply:
Done